pkgs/servers/geospatial/martin/default.nix

```diff
@@ -8,12 +8,17 @@ rustPlatform.buildRustPackage rec {
     owner = "urbica";
     repo = pname;
     rev = "v${version}";
-    sha256 = "1i9zhmjkgid4s90n52wqmrl3lwswcaxd6d47ssycgjl1nv0jla4k";
+    hash = "sha256-kygqwbaByse81oc007piXHM6aK6Yi2JB0qTFN2WFP8U=";
   };
 
-  cargoSha256 = "1jgl8s6h4pqhn189swrhn896kw1rkmqpf7dq52ry2rdci80g02nq";
+  cargoPatches = [
+    # Remove after a new release, tracked by https://github.com/maplibre/martin/issues/410.
+    ./update-socket2-for-rust-1.64.patch
```

Comment:
We can't fetch this?

Reply:
I couldn't figure out how to do it, because this repository has [so many automated dependency bump commits](https://github.com/maplibre/martin/commits/main) that shift the line numbers in `Cargo.lock`.

Reply:
Then we do it like this.